rllib/env/tests/test_multi_agent_env.py

```diff
@@ -318,7 +319,7 @@ def test_multi_agent_sample_round_robin(self):
         )
 
     def test_custom_rnn_state_values(self):
-        h = {"some": {"arbitrary": "structure", "here": [1, 2, 3]}}
+        h = {"some": {"here": np.array([1.0, 2.0, 3.0])}}
```

Comment:
Having text in state is not a realistic scenario that will cause some issue in the recursive unbatching. I fixed it like this.